plugin/plugin.go

```diff
@@ -28,7 +28,7 @@ type Registration struct {
 	Init   func(*InitContext) (interface{}, error)
 }
 
-// TODO(@crosbymichael): how to we keep this struct from growing but support dependency injection for loaded plugins?
+// TODO(@containerd): how to we keep this struct from growing but support dependency injection for loaded plugins?
```

Comment:
don't think you intended this, unless you don't want your name associated with it ;)